new/language/implementation/javascript/javascript.go

```diff
@@ -239,6 +239,10 @@ func (implementation *javascriptImplementation) IsRootOfRuleQuery(node *tree.Nod
 }
 
 func (implementation *javascriptImplementation) PatternNodeTypes(node *tree.Node) []string {
+	if node.Type() == "statement_block" && node.Parent().Type() == "program" {
```

Comment:
I think we should also check for children, that will allow us to still match against statement blocks in patterns:

```suggestion
	if node.Type() == "statement_block" && node.Parent().Type() == "program" && node.NamedChildCount() == 0 {
```